src/ui.form-radio/types.ts

```diff
@@ -0,0 +1,97 @@
+import defaultConfig from "./config.ts";
+
+import type { UnknownObject, UnknownArray } from "../types.ts";
+
+export type Config = Partial<typeof defaultConfig>;
+
+export interface URadioProps {
+  /**
+   * Radio value.
+   */
+  modelValue?: boolean | string | number | UnknownArray | UnknownObject;
+
+  /**
+   * Native value attribute.
+   */
+  value?: boolean | string | number | UnknownArray | UnknownObject;
+
+  /**
+   * Radio label.
+   */
+  label?: string;
+
+  /**
+   * Label placement.
+   */
+  labelAlign?: "left" | "right";
+
+  /**
+   * Radio description.
+   */
+  description?: string;
+
+  /**
+   * Error message.
+   */
+  error?: string;
+
+  /**
+   * Radio name.
+   */
+  name?: string;
+
+  /**
+   * Radio size.
+   */
+  size?: "sm" | "md" | "lg";
+
+  /**
+   * Radio color.
+   */
+  color?:
```

Comment:
set colors in the same order as in UButton (applies to all components in this PR)